APIClient/Program.cs

```diff
@@ -7,18 +7,32 @@
 
 namespace ProductStoreClient
 {
+    static class Constants
+    {
+
+        public const string ApiUrl = "https://localhost:7016/";
```

Comment:
Remember this:
by convention we expect the domain without a trailing slash
and the endpoint with a starting slash